core/bootstrap.go

```diff
@@ -202,7 +201,7 @@ func bootstrapConnect(ctx context.Context, ph host.Host, peers []peer.PeerInfo)
 	return nil
 }
 
-func toPeerInfos(bpeers []config.BootstrapPeer) []peer.PeerInfo {
+func toPeerInfos(bpeers []config.BootstrapPeer) ([]peer.PeerInfo, error) {
```

Comment:
oh whoops... I was using the `iaddr.ParseMultiaddr` to parse the `/ipfs/Qmasdasd` part off of the addresses, then i realized that was already being done for me.